html/elements/meta.json

```diff
@@ -124,6 +124,469 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "height": {
+            "__compat": {
+              "description": "content=\"height=[value]\"",
+              "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Viewport_meta_tag#height",
+              "spec_url": "https://drafts.csswg.org/css-viewport/#viewport-meta",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": "18"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": "4"
+                },
```

Comment:
Same question, the [Firefox 4 release notes ](https://blog.mozilla.org/en/mozilla/mozilla-launches-firefox-4-for-android-allowing-users-to-take-the-power-and-customization-of-firefox-everywhere-2/), including the [full release notes](https://website-archive.mozilla.org/www.mozilla.org/firefox_releasenotes/en-us/mobile/4.0/releasenotes/), don't seem to mention these features?